### ISSUE TITLE
Etapa 3: duda selectores de mux y demux a agregar.

### ISSUE THREAD

**Issue by brayan788** [IIC2343/Syllabus-2023-2#71]
Hola!!,  no me queda claro del diagrama como el ram_adress hace de selector en el mux_in y demux_out o cuales son los bits que se ocupan de este para seleccionar en ambos casos?

**Comment by frvalenzuela** [comment]
Hola @brayan788 !

En ambos casos se ocupa las siguientes direcciones:

![image](https://github.com/IIC2343/Syllabus-2023-2/assets/20147482/83714166-bb0a-4c1e-b58f-7a8c6721996e)

En el caso de MuxIn puedes asumir que si no son una de esas direcciones, entonces lo que entra es lo que sale de la RAM. En caso de Demux si no es una de esas direcciones no llega señal hacia los otros componentes.

Saludos cordiales!

**Comment by brayan788** [comment]
un ejemplo para ver si entendi bien si ram_adress es "000000000000" entonces el mux_in selecciona los leds?

**Comment by frvalenzuela** [comment]
No, porque el Mux_in es un dispositivo de entrada. Los leds es un dispositivo de salida, por lo que será con el Demux.

Saludos!

**Comment by brayan788** [comment]
aah cierto, pero entiendo la idea gracias por la rápida respuesta.
